src/next.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = withCSS({
       // Make some environment variables accessible from the client
       new webpack.EnvironmentPlugin({
         OPENCOLLECTIVE_REFERRAL: null,
+        OPENCOLLECTIVE_REDIRECT_URL: process.env.OPENCOLLECTIVE_REDIRECT_URL,
```

Comment:
Was that tested?

Reply:
Sorry, I had `OPENCOLLECTIVE_REDIRECTION_URL` instead of `OPENCOLLECTIVE_REDIRECT_URL`.

Anyway, here, just put `null` like `OPENCOLLECTIVE_REFERRAL`, this is the default value.

Reply:
Don't forget that.